.travis.yml

```diff
@@ -5,3 +5,5 @@ install:
     - true
 script:
     - mvn verify
+    - mvn clean verify -Pfunctional-tests
```

Comment:
I am not clear, what should be done here, should we add multiple profiles to the `mvn verify` command ?